README.md

```diff
@@ -67,6 +67,10 @@ CarbonData is built using Apache Maven, to [build CarbonData](https://github.com
 * [Carbon as Spark's Datasource](https://github.com/apache/carbondata/blob/master/docs/carbon-as-spark-datasource-guide.md) 
 * [FAQs](https://github.com/apache/carbondata/blob/master/docs/faq.md) 
 
+##  Integrations
+* [Hive](https://github.com/apache/carbondata/blob/master/docs/hive-guide.md)
+* [Presto](https://github.com/apache/carbondata/blob/master/integration/presto/Presto_Cluster_Setup_For_Carbondata.md)
```

Comment:
Can you move the doc to docs folder?

Reply:
OK,I have unify it i,and move the doc to the docs folder and rename the doc to the presto-guide